server/app_env/custom.py

```diff
@@ -23,6 +23,8 @@ class Config(_base.Config):
   LOGO_PATH = "/custom_dc/custom/logo.png"
   SHOW_DISASTER = False
   LITE = True
+  USE_PALM = False
+  USE_MEMCACHE = False
```

Comment:
No need to set since these are default?

Reply:
Perhaps good to be explicit about it in case the defaults change?

Reply:
sg